sharedresource/v1alpha1/0000_10_sharedresource.crd.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/979
+    description: Extension for sharing ConfigMaps and Secrets across Namespaces
+    displayName: SharedResource
+  name: sharedresources.storage.openshift.io
+spec:
+  group: storage.openshift.io
```

Comment:
@coreydaley this looks like the old CRD - did you forget to remove this?

Reply:
yep, should be gone now.